modules/raft/src/main/java/org/apache/ignite/internal/raft/storage/impl/VolatileLogStorageFactory.java

```diff
@@ -118,6 +121,18 @@ public LogStorage createLogStorage(String groupId, RaftOptions raftOptions) {
         return new VolatileLogStorage(createLogStorageBudget(), new OnHeapLogs(), spiltOnDisk);
     }
 
+    @Override
+    public boolean destroyLogStorage(String uri) {
+        try {
+            RocksDbSpillout.deleteAllEntriesBetween(db, columnFamily, groupStartPrefix(uri), groupEndPrefix(uri));
+            return true;
+        } catch (RocksDBException e) {
+            LOG.error("Fail to destroy the log storage spillout for {}.", e, uri);
```

Comment:
Same about exception.

Maybe then add some kind of handle to the user so that he can then manually try to delete it again, or should we somehow automatically try again?